kaizen/autofix/test/test_framework_integration.py

```
@@ -0,0 +1,55 @@
+""")
+        test_file = f.name
+    
+    try:
+        workspace_root = Path(tempfile.gettempdir())
+        executor = CodeRegionExecutor(workspace_root)
+        
+        # Create region info
+        region_info = RegionInfo(
+            type=RegionType.CLASS,
+            name='TestAgent',
+            code=open(test_file).read(),
```

Ruff (0.11.9)

Use a context manager for opening files

(SIM115)